### ISSUE TITLE
Pretraining/training script, loss functions for hi-ml-multimodal (BioViL)  

### ISSUE THREAD

**Issue by santosh9sanjeev** [microsoft/hi-ml#894]
Hello! Thanks for the wonderful work and for sharing the pretrained weights.

In your ECCV work (`Making the Most of Text Semantics to Improve Biomedical Vision–Language Processing`), there are several results on classification and segmentation(zero-shot) tasks.
I wanted to get the **pretraining script** which has information about the loss functions, hyperparameter settings etc. , but I am not able to find it.

Can you provide some information regarding where we can get these **training scripts**?
Thank you!

**Comment by sayeh1994** [comment]
Following.

**Comment by darkstar1227** [comment]
Is that possible I can train my own custom data?
